### PR TITLE
Add support for nullable fields using Nullable

### PR DESCRIPTION
**Pull request by nickwilliams-eventbrite**

### CODE REVIEW

**Comment by ebwill** [comment]
Looks good to me. Would be cool to see a `NullableUnicodeString`, but we can add in separate diff

**Comment by nickwilliams-eventbrite** [comment]
>Looks good to me. Would be cool to see a NullableUnicodeString, but we can add in separate diff

@ebwill, you would simply do that like this: `Nullable(UnicodeString())`.

**Comment by ebwill** [comment]
oh! Nifty 👍

File: conformity/fields/meta.py [hunk 1]
```diff
@@ -8,6 +8,28 @@
...
+
+    def errors(self, value):
+        if value is None:
+            return []
+
+        return self.field.errors(value)
+
+    def introspect(self):
+        return strip_none({
+            'type': 'nullable({})'.format(self.field.introspect),
```
Comment (andrewgodwin) [inline conformity/fields/meta.py]: Please use double quotes for consistency with the rest of the library.

File: tests/test_fields_meta.py [hunk 2]
```diff
@@ -20,6 +22,25 @@ class MetaFieldTests(unittest.TestCase):
     Tests meta fields
     """
 
+    def test_nullable(self):
+        schema = Nullable(Constant('one', 'two'))
```
Comment (andrewgodwin) [inline tests/test_fields_meta.py]: Double quotes please.

File: conformity/fields/meta.py [hunk 3]
```diff
@@ -162,7 +184,7 @@ def errors(self, value):
             ok = self.validator(value)
         except Exception:
             return [
-                Error("Validator errored (invalid type?)"),
+                Error("Validator erred (invalid type?)"),
```
Comment (vartec) [inline conformity/fields/meta.py]: If we're going deeply grammatical, then it's incorrect. Validator didn't _err_, it encountered an _error_.

File: conformity/fields/meta.py (hunk 1 above)
Comment (vartec) [inline conformity/fields/meta.py]: So Nullable allows `None`, yet it won't prevent inner `self.field.introspect()` from using `strip_none`

BTW `field.introspect` is not `@property` so it needs to be called

Comment (nickwilliams-eventbrite) [reply conformity/fields/meta.py]: Fair.

File: tests/test_fields_meta.py (hunk 2 above)
Comment (nickwilliams-eventbrite) [reply tests/test_fields_meta.py]: Fair.

File: conformity/fields/meta.py (hunk 1 above)
Comment (nickwilliams-eventbrite) [reply conformity/fields/meta.py]: Fixed.

File: conformity/fields/meta.py (hunk 3 above)
Comment (nickwilliams-eventbrite) [reply conformity/fields/meta.py]: Changed.

**Pull request merged**